tao_compiler/mlir/disc/transforms/disc_shape_optimization_utils.cc

```diff
@@ -451,26 +517,33 @@ LogicalResult SymbolicDimMgr::save() {
           (llvm::Twine("S") + llvm::Twine(numNonConstDims++)).str();
     }
```

Comment:
`en.index()` not used, no need to use `llvm::enumerate`?

Reply:
done